### PR TITLE
[IT] Translated week03 intro + lecture01

### PR DESCRIPTION
**Pull request by marcozullich**

### CODE REVIEW

**Comment by skyil7** [comment]
Please Let us know which language you translated into.

Use our naming convention.
[Your Language code]Translated week03 intro + lecture01

File: docs/it/week03/03-1.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
+lang: it
+lang-ref: ch.03-1
+title: Visualizzazione delle trasformazioni dei parametri delle reti neurali e concetto fondamentale di convoluzione
+data: 10 febbraio 2020
+data traduzione: 2 aprile 2020
```
Comment (Atcold) [inline docs/it/week03/03-1.md:6]:
```suggestion
translation-date: 2 Apr 2020
```

File: docs/it/week03/03-1.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
+---
+lang: it
+lang-ref: ch.03-1
+title: Visualizzazione delle trasformazioni dei parametri delle reti neurali e concetto fondamentale di convoluzione
+data: 10 febbraio 2020
```
Comment (Atcold) [inline docs/it/week03/03-1.md:5]:
```suggestion
date: 10 Feb 2020
```

File: docs/it/week03/03-1.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
+---
+lang: it
+lang-ref: ch.03-1
+title: Visualizzazione delle trasformazioni dei parametri delle reti neurali e concetto fondamentale di convoluzione
+data: 10 febbraio 2020
+data traduzione: 2 aprile 2020
+traduttore: marcozullich
```
Comment (Atcold) [inline docs/it/week03/03-1.md:7]:
```suggestion
translator: marcozullich
```

File: docs/it/week03/03.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+lang-ref: ch.03
```
Comment (Atcold) [inline docs/it/week03/03.md:2]: `lang: it` is missing.

File: docs/it/week03/03.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+titolo: Week 3
```
Comment (Atcold) [inline docs/it/week03/03.md:3]: This is going to break our website.
There's a program reading these headers.
You cannot change the name of the variables! LOL

File: docs/it/week03/03.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+data traduzione: 30 marzo 2020
+traduzione: marcozullich
```
Comment (Atcold) [inline docs/it/week03/03.md:5]: Same for these two.
These are like `for`, `while`, `if` in a C program.
You cannot change them, haha.

**Comment by Atcold** [comment]
@skyil7, use the *Review* function to enforce a change, so that the merging is suspended until such change is performed.

**Comment by Atcold** [comment]
@marcozullich, you need to apply the corrections I've pointed out before I can merge.
Do you see these messages at all?

**Comment by Atcold** [comment]
You cannot use `titolo:` since the program that is creating the `HTML` pages from the `MD` one does not speak Italian. It's a script that is using the entries in order to populate the correct fields. You cannot translate them!

**Comment by marcozullich** [comment]
@Atcold I corrected them with commit c96ab1d but somehow older files from another branch came back into master, I have to look into that and correct

**Comment by marcozullich** [comment]
With commit 8b313c8 everything should be ok with headers

File: docs/it/week03/03.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+---
+lang: it
+lang-ref: ch.03
+title: Week 3
```
Comment (Atcold) [inline docs/it/week03/03.md:4]:
```suggestion
title: Settimana 3
```

File: docs/_config.yml [hunk 8]
```diff
@@ -167,3 +167,6 @@ it:
         - path: it/week01/01-2.md
         - path: it/week01/01-3.md
     - path: it/week02/02.md
+    - path: it/week03/03.md
+      sections:
+          -path: it/week03/03-1.md
```
Comment (Atcold) [inline docs/_config.yml:172]: And here you broke the website (there is no space between `-` and `path`). LOL

**Pull request merged**